locales/en-US/app.ftl

```diff
@@ -122,6 +122,11 @@ retirePageHeadline = Thanks for flying!
 retirePageMessage = Hope you had fun experimenting with us. <br> Come back any time.
 retirePageSurveyButton = Take a quick survey
 
+restartIntroLead = Preflight checklist
+restartIntroOne = 1. Restart your browser
```

Comment:
We should be putting this in an ordered list rather than putting the bullets here.